README.md

```diff
@@ -117,6 +118,40 @@ const sql = SQL`INSERT INTO users (id, name) VALUES
 `
 ```
 
+### map(array, mapperFunction)
+
+Using the default mapperFunction which is just an iteration over the array elements
+```js
+const ids = [1, 2, 3]
+
+const values = SQL.map(ids)
+const sql = SQL`INSERT INTO users (id) VALUES (${values})`
+```
+
+Using an array of objects which requires a mapper function
+
+```js
+const objArray = [{
+  id: 1,
+  name: 'name1'
+},
+{
+  id: 2,
+  name: 'name2'
+},
+{
+  id: 3,
+  name: 'name3'
+}]
+
+const mapperFunction = (objItem) => {
+  return objItem.id
+}
```

Comment:
why don't you put this inline? I think it would be a more common use case

Reply:
Yepp, and it's easier to read. Pushed a commit.